Formula/fish.rb

```diff
@@ -39,7 +54,30 @@ def caveats; <<-EOS.undent
     EOS
   end
 
+  def post_install
+    (share/"fish/vendor_functions.d").mkpath
```

Comment:
@dunn We already discussed that, check your comments earlier.
using pkgshare defeats purpose of this PR.
I use share to have system-wide directory so other formulas can put files in that dir .